contracts/HydraChain/HydraChain.sol

```diff
@@ -112,8 +109,6 @@ contract HydraChain is
         _commitBlockNumbers[newEpochId] = block.number;
         epochEndBlocks.push(epoch.endBlock);
 
-        _applyPendingExp(); // Apply new exponent in case it was changed in the latest epoch
```

Comment:
Why do we remove the pedningExp functionality, isn't it right to update the exponent on the epoch commit?

Reply:
The node uses the event to intercept the change and it updates the validator powers based on the change in the power exponent at the end of the epoch, so in practice, the apply mechanism is moved to the node.